app/src/components/forms/AddTeamForm.vue

```diff
@@ -122,10 +142,58 @@ const team = defineModel({
     members: [{ name: '', address: '', isValid: false }]
   }
 })
+
+const customValidators = {
+  isValidAddress: (value: any) => {
+    return isAddress(value)
+  }
+}
+
+const rules = {
+  team: {
+    name: { required },
+    members: {
+      $each: helpers.forEach({
+        address: {
+          required,
+          isValid: helpers.withMessage('Invalid address', customValidators.isValidAddress)
+        }
+      })
+    }
+  }
+}
+
+const $v = useVuelidate(rules, { team })
```

Comment:
This might introduce conflicts don't you think?
The $ prefix is reserved for internal Vue utilities.